contents/docs/product-analytics/sharing.mdx

```diff
@@ -1,22 +1,44 @@
 ---
 title: Sharing or embedding insights
 ---
+import shareOrEmbedLight from '../../images/docs/product-analytics/share-or-embed-light-mode.png'
+import shareOrEmbedDark from '../../images/docs/product-analytics/share-or-embed-dark-mode.png'
+import sharePubliclyLight from '../../images/docs/product-analytics/share-publicly-light-mode.png'
+import sharePubliclyDark from '../../images/docs/product-analytics/share-publicly-dark-mode.png'
+import embedInsightLight from '../../images/docs/product-analytics/embed-insight-light-mode.png'
+import embedInsightDark from '../../images/docs/product-analytics/embed-insight-dark-mode.png'
 
 You can share a public link to a specific insight or embed an insight in a web page or application (where the insight will be displayed in an iframe).
 
 ## Share an insight's public link
 
-1. Choose _Share or embed_ from the `...` (more options) menu.
+1. Choose _Share or embed_ from the `...` (more options) menu located at the top right corner of the page.
 
-  ![Sharing an insight via the PostHog app](../../images/docs/product-analytics/share-or-embed.png)
+<ProductScreenshot
```

Comment:
Can we replace this image with one that looks better (wider)? You can add an arrow if it helps. No matter if this image is center or not it will look bad.